reflectutil/setter.go

```diff
@@ -132,6 +132,13 @@ func valueFor(typ reflect.Type, tag reflect.StructTag, value interface{}) reflec
 				list = reflect.Append(list, valueFor(typ.Elem(), tag, elt))
 			}
 		case *protop.Literal:
+			if typ.Elem().Kind() == reflect.TypeOf(byte(0)).Kind() && value.IsString {
```

Comment:
```
if etyp := typ.Elem(); etyp.Kind() == reflect.Uint8 && value.IsString {
```

(and reuse `etyp` below)

Reply:
Or even better, declare `etyp` at the top of the `case reflect.Slice:`

Reply:
> Or even better, declare etyp at the top of the case reflect.Slice:

I did that in latest commit.